src/support_code_library_builder/types.ts

```diff
@@ -69,6 +69,8 @@ export interface IParameterTypeDefinition<T> {
   preferForRegexpMatch?: boolean
 }
 
+export type IPublishedHandler = (report: { url: string }) => void
```

Comment:
Made the argument an object with a `url` property, envisaging that someday there might be more available e.g. time to expiry etc.